package/gluon-web-admin/luasrc/lib/gluon/web/model/admin/remote.lua

```diff
@@ -30,11 +31,17 @@ function keys:write(value)
 	end
 end
 
+local min_password_length = ((site.config_mode or {}).remote_login or {}).min_password_length or 12
```

Comment:
But we try to fix an existing serious deficit here, so adding a minimum length is cruicial to security in Freifunk.

And there is no drawback for users, that used a short password before, because there cannot be no check of en existing password that is too short.